scripts/cli/src/helpers/rewards_helper.ts

```diff
@@ -0,0 +1,36 @@
+import type { KeyringPair } from "@polkadot/keyring/types";
+import type { AccountId } from "@polkadot/types/interfaces/runtime";
+import { sendTx, ApiSingleton } from "../util/init";
+import { ItnRewardStatus } from "../types";
+import { u8aToHex, numberToU8a } from "@polkadot/util";
+
+/**
+ * @description Claim an ITN reward with a valid signature.
+ */
+export async function claimItnReward(
+	signer: KeyringPair,
+	itnAddress: string | Uint8Array | AccountId
+) {
+	const api = await ApiSingleton.getInstance();
+	const rewardAddress = signer.publicKey;
+	let signature = (u8aToHex(rewardAddress, 512)).toString();
+	signature = signature + "claim_itn_reward";
+	const transaction = api.tx.rewards.claimItnReward(rewardAddress, itnAddress, {
+		Sr25519: signature,
+	});
+	await sendTx(signer, transaction);
```

Comment:
This needs to be submitted unsigned.  That `signer` isn't needed here, just the transaction.